### DISCUSSION TITLE
Documentation: inlet.snmp.agents

### DISCUSSION THREAD

**Discussion by sm3ulc** [akvorado/akvorado#299] [Q&A] [answered]
Trying to get Akvorado running with a softflowd-exporter and no SNMP is possible. Please give an example how to use **inlet.snmp.agents**

**Comment by vincentbernat** [comment, marked as answer]
it is not possible to use Akvorado without SNMP. You can install a SNMP daemon on your exporter.

**Reply by sm3ulc** [reply]
With SNMP in the exporter running softflowd and maybe found more problems. The index referred from softflowd is **'0'**  as in **'undefined'**.

**snmpwalk -v 2c -c public 1.2.3.4 .**
....
IF-MIB::ifIndex.1 = INTEGER: 1
IF-MIB::ifIndex.2 = INTEGER: 2
**IF-MIB::ifIndex.3 = INTEGER: 3**
IF-MIB::ifIndex.6 = INTEGER: 6
IF-MIB::ifIndex.294 = INTEGER: 294
IF-MIB::ifIndex.645 = INTEGER: 645
IF-MIB::ifIndex.646 = INTEGER: 646
IF-MIB::ifIndex.651 = INTEGER: 651

softflowd is running on **ifIndex.3**

**From the flow via wireshark:**

Flow 1
    SrcAddr: 1.2.3.4
    DstAddr: 4.5.6.8
    [Duration: 0.000000000 seconds (switched)]
    Octets: 126
    Packets: 1
    **InputInt: 0
    OutputInt: 0**
    SrcPort: 46976
    DstPort: 49420
    Protocol: UDP (17)
    TCP Flags: 0x00
    IPVersion: 4
    IP ToS: 0x00

**Reply by vincentbernat** [reply]
It seems that by default, softflowd does not include the interface. It is configurable as another user was able to make it work (search for `softflowd` in discussions).

**Comment by sm3ulc** [comment]
Thanks for the info. SNMP is a great enrichment bit I would advise you to make it possible to run without SNMP. More or less all other netflow-software does not require SNMP and with network-taps interface information makes no sense.
